README.md

```diff
@@ -79,3 +77,45 @@ terraform destroy -var-file=./input.tfvars
 
 All optional parameters, by default, will be set to `null` in respective example's variable.tf file. You can also override these optional parameters.
 
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+## Inputs
+
```

Comment:
Add following hook in validate.yml github action (Take code from https://github.com/terraform-ibm-modules/terraform-ibm-module-template repo)

auto-update hook

Reply:
done